test/e2e/scheduling/priorities.go

```diff
@@ -466,6 +466,8 @@ func updateNodeAllocatable(c clientset.Interface, nodeName string, memory, cpu i
 	if err != nil {
 		return err
 	}
+	node.Status.Capacity[v1.ResourceMemory] = *resource.NewQuantity(memory, resource.BinarySI)
```

Comment:
Two concerns:

1. The function is named as `updateNodeAllocatable`, but the logic inside is updating `Capacity` as well. It should be consistent.
1. It seems the **capacity** field is not set back to original value after testing?

Reply:
Yup, both are right, I am going to address them.